frontend/ui/widget/menu.lua

```diff
@@ -963,6 +963,7 @@ function Menu:init()
     if Device:hasKeys() then
         -- set up keyboard events
         self.key_events.Close = { { Input.group.Back } }
+        self.key_events.LeftButtonTap = { { "Menu" } }
```

Comment:
`LeftButtonTap` becomes a bad name if you add this.

Reply:
What do you mean by "becomes a bad name"? That is what the existing function is called

Reply:
It's changing the original context in which it was named, making it confusing. Something like `WidgetMenuToggle`? Pinging @hius07 who wrote it for some thoughts.

Reply:
If it works, I think it's okay.